guard/src/main.rs

```diff
@@ -62,7 +64,10 @@ fn main() -> Result<(), Error> {
 
```

Comment:
Should be somewhere before this block, we initialize Stdout to be the err buffer, right?